<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mapDialog</class>
 <widget class="QDialog" name="mapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>160</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Map Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hlP1">
     <item>
      <widget class="QLabel" name="lblP1">
       <property name="text">
        <string>P1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leP1FileLocation">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>File Location</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnP1LoadFile">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Load File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlP2">
     <item>
      <widget class="QLabel" name="lblP2">
       <property name="text">
        <string>P2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leP2FileLocation">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>File Location</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnP2LoadFile">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Load File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlBtn">
     <item>
      <widget class="QPushButton" name="btnLoadExpl">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Load map for exploration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoadFastPath">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Load map for Fastest Path</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
